### PR TITLE
refactor(wp): send many uniform error messages changes

### PR DESCRIPTION
**Pull request by krupanand-bitgo**
Standardized error messages for the Bulk Withdrawal API across different assets to ensure consistency.
Ticket: COIN-1957

### CODE REVIEW

**Review by alebusse — DISMISSED** [review]

**Review by mukeshsp — DISMISSED** [review]

**Review by zahin-mohammad — COMMENTED** [review]
Do we need this pr and https://github.com/BitGo/bitgo-microservices/pull/44632 ??

File: modules/sdk-coin-bera/test/unit/bera.ts [hunk 1]
```diff
@@ -302,7 +302,7 @@ describe('Bera', function () {
 
       await basecoin
         .verifyTransaction({ txParams, txPrebuild, wallet, verification })
-        .should.be.rejectedWith('txPrebuild should only have 1 recipient but 2 found');
```
Comment (zahin-mohammad) [inline modules/sdk-coin-bera/test/unit/bera.ts:305]: Update code owners for this file. 

Wallets team does not own the coin specific tests.

Comment (krupanand-bitgo) [reply modules/sdk-coin-bera/test/unit/bera.ts:305]: Added 
/modules/sdk-coin-bera/ @BitGo/ethalt-team

**Review by therealdwright — DISMISSED** [review]

**Comment by krupanand-bitgo** [comment]
> Do we need this pr and [BitGo/bitgo-microservices#44632](https://github.com/BitGo/bitgo-microservices/pull/44632) ??

Yes.
There changes required in both BGMS and Bitgo JS

File: modules/abstract-eth/test/unit/coin.ts [hunk 2]
```diff
@@ -348,7 +348,9 @@ export function runTransactionVerificationTests(coinName: string, bitgo: TestBit
 
       await basecoin
         .verifyTransaction({ txParams, txPrebuild, wallet, verification })
-        .should.be.rejectedWith('txPrebuild should only have 1 recipient but 2 found');
+        .should.be.rejectedWith(
+          `${basecoin} doesn't support sending to more then 1 destination address within a single transaction. Try again, using only a single recipient.`
```
[the remark this answers is not in the document]
Comment (krupanand-bitgo) [reply modules/abstract-eth/test/unit/coin.ts]: Yeah, have to pass coinName, done the changes accordingly.
Thanks for pointing.

**Review by zahin-mohammad — DISMISSED** [review]

**Review by mukeshsp — APPROVED** [review]

**Review by margueriteblair — APPROVED** [review]

**Review by alebusse — APPROVED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Review by therealdwright — APPROVED** [review]

**Pull request merged**